src/css/typography.css

```diff
@@ -41,12 +42,16 @@
   a,
   p > a,
   ul > li > a {
-    @apply text-href-base underline;
-  }
-
-  a:hover,
-  a:active {
-    @apply text-href-hover;
+    background-image: linear-gradient(0deg, #f56565, #f56565);
```

Comment:
Oh, forgot to check whether I can get `.text-red-500` here, hardcoded it whilst experimenting